corehq/ex-submodules/couchforms/getters.py

```diff
@@ -30,6 +32,11 @@ def get_instance_and_attachment(request):
     if request.META['CONTENT_TYPE'].startswith('multipart/form-data'):
         # ODK submission; of the form
         # $ curl --form 'xml_submission_file=@form.xml' $URL
+
+        if request.POST.keys():
+            notify_error("Received a submission with POST.keys()")
+            raise Exception("Received a submission with POST.keys()")
```

Comment:
Thoughts on raising a more specific error here, and then trapping it higher up the chain and responding with a `BadRequest` or something less loud?